src/tests/index.test.ts

```diff
@@ -6,6 +6,12 @@ describe("ad to bs testing", () => {
     // expect(result).toHaveProperty("year", 2053);
     expect(result).toMatchObject({ year: 2053, month: 6, day: 25 });
   });
+
+  it("should return current date", () => {
+    const result = adToBS(Date.now());
+    expect(result).toHaveProperty("year", 2081);
```

Comment:
this will fail next year. you should really stub out the `Date` object or use a fake one instead. even using a known static value is fine in this particular case